### PR TITLE
Adds more 513 compat stuff

### PR DESCRIPTION
**Pull request by spookydonut**
## About The Pull Request
islist()  tan()  clamp()  ismovable() are all new built-ins
stuff like TAN(X) should maybe just be find and replaced with tan(x) then change the macro to be 
```
#if DM_VERSION < 513
#define tan(x) (sin(x) / cos(x))
#endif
```

### CODE REVIEW

File: code/__DEFINES/is_helpers.dm [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 
+#if DM_VERSION < 513
 #define islist(L) (istype(L, /list))
+#endif
 
 #define in_range(source, user) (get_dist(source, user) <= 1 && (get_step(source, 0)?:z) == (get_step(user, 0)?:z))
 
-#define ismovableatom(A) (istype(A, /atom/movable))
+#if DM_VERSION < 513
+#define ismovable(A) (istype(A, /atom/movable))
```
Comment (ninjanomnom) [inline code/__DEFINES/is_helpers.dm]: We don't want the new ismovable probably, make sure it doesnt have the same problem isatom and ismob have

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Comment by ninjanomnom** [comment]
I'd rather have these mass of changes get done when we're actually using 513, it's harmless to use the old versions until then

File: code/__DEFINES/is_helpers.dm (hunk 1 above)
Comment (optimumtact) [reply code/__DEFINES/is_helpers.dm]: what problem

Comment (spookydonut) [reply code/__DEFINES/is_helpers.dm]: what are those problems?

Comment (ninjanomnom) [reply code/__DEFINES/is_helpers.dm]: oh wait right it was isobj, the native isobj acted like ismovable before so we overrode it. you should probably just check if ismovable works correctly compared to our current version

This is probably fine though

**Pull request merged**